datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/mirrormaker/KafkaMirrorMakerConnectorTask.java

```diff
@@ -103,9 +104,11 @@
   public static final String DOMAIN_TOPIC_MANAGER = "topicManager";
   public static final String TOPIC_MANAGER_METRICS_PREFIX = "TopicManager";
 
+  protected final DynamicMetricsManager _dynamicMetricsManager;
```

Comment:
Do you need the new variables to be protected? I dont see their usage outside this class.

Reply:
Yeah, I made these protected as we are extending these to add Audit V3 support.